.vscode/settings.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Should give us shared vscode settings for things like eslint (it was ignored)

Reply:
👍